docs/control-structures.rst

```diff
@@ -18,12 +18,17 @@ Functions are executable units of code within a contract. Functions may only be
 
 Functions may be called internally or externally depending on their :ref:`visibility <function-visibility>`. Functions may accept input arguments and return variables in order to pass values between them.
 
-.. _function-visibility:
-
 Visibility
 ----------
 
-All functions must include exactly one visibility decorator.
+.. _function-visibility:
+
+You can optionally declare a function's visibility by using a :ref:`decorator <function-decorators>`. There are three visibility levels in vyper:
+
+    * **External**: exposed in the selector table, can be called by an external call into this contract
```

Comment:
```suggestion
    * ``@external``: exposed in the selector table, can be called by an external call into this contract
```

Reply:
it's consistent with the section on mutability

Reply:
hmm, yeah I don't like that section either. If I read the docs of a programming language, I want to learn about the syntax specifically, so I would suggest to refactor the mutability section as well.

Reply:
09231da8b581e476fe5a0af0866af5e965b06c0e